.github/workflows/git-checks.yml

```diff
@@ -0,0 +1,12 @@
+name: Git Checks
```

Comment:
Nitpick: let's use a more specific name (for now)

```suggestion
name: Block !fixup commits
```

Reply:
I thought about it, but let it on purpose, so we can reuse this for more git checks.

The status check is clear enough:

![imagen](https://github.com/user-attachments/assets/b179232f-c8b6-460d-a843-ff4297addb4b)

But if you want I can change the name of the step to `block-fixup-commits`

Reply:
In the end I can easily see us needing some more checks after we end up forcing GPG.

Reply:
Fair enough. I don't have a strong opinion.